CRT.h

```diff
@@ -158,6 +158,9 @@ typedef enum ColorElements_ {
    DYNAMIC_MAGENTA,
    DYNAMIC_YELLOW,
    DYNAMIC_WHITE,
+#ifdef NVIDIA_JETSON
+   GPU_FILTER,
+#endif
```

Comment:
No need to define this conditional. Also, probably group near the `GPU_ENGINE_x` fields.